### PR TITLE
Create unit tests

### PR DESCRIPTION
**Pull request by BRuslanB**
Create 23 unit tests in Package kz.bars.familybudget.dto.mapping and kz.bars.familybudget.service.impl

### CODE REVIEW

File: src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+		budgetRepo.save(budget);
+
+		Budget currentBudget = budgetService.getBudget(budget.getId());
+		Assertions.assertNotNull(currentBudget);
+		Assertions.assertEquals(currentBudget.getId(), budget.getId());
+		Assertions.assertEquals(currentBudget.getIncome(), budget.getIncome());
+		Assertions.assertEquals(currentBudget.getDescription(), budget.getDescription());
+
+		budgetRepo.deleteAll();
```
Comment (btimur) [inline src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java:105]: delete/clear  oppertaion better to move separte  method with annotation @After

File: src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	}
+
+	@Test
+	public void checkBudgetGetAll() {
+		budgetRepo.deleteAll();
+
+		List<Budget> budgetList = new ArrayList<>();
```
Comment (btimur) [inline src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java:112]: this init DB with predeafilt values can be moved or to @beforeAll method or  to data Init script

File: src/test/resources/application-local.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#server:
+#  port: 8080
+
+spring:
```
Comment (btimur) [inline src/test/resources/application-local.yml:4]: u don need this file overriding in test folder

File: src/test/resources/application.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#server:
+#  port: 8080
+
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:2345/family_budget_test_db
```
Comment (btimur) [inline src/test/resources/application.yml:6]: for test usually u can use in memory DB  like HBASE with PostgreSQL diaelct

File: src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java (hunk 1 above)
Comment (BRuslanB) [reply src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java:105]: I agree, I added methods with annotation @AfterEach for class tests

File: src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java (hunk 2 above)
Comment (BRuslanB) [reply src/test/java/kz/bars/familybudget/service/impl/BudgetServiceTests.java:112]: I agree, I added methods with annotation @BeforeEach for class tests

File: src/test/resources/application-local.yml (hunk 3 above)
Comment (BRuslanB) [reply src/test/resources/application-local.yml:4]: I agree, I deleted record "server:port:8080"

File: src/test/resources/application.yml (hunk 4 above)
Comment (BRuslanB) [reply src/test/resources/application.yml:6]: I added the project H2 Database for Unit Tests

**Pull request merged**